.evergreen/config.yml

```diff
@@ -51,6 +51,11 @@ functions:
               export PROJECT_DIRECTORY=$(cygpath -m $PROJECT_DIRECTORY)
            fi
 
+           # Installation of cryptography requires a rust compiler on some machines
+           curl https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path
+           export PATH="$HOME/.cargo/bin:$PATH"
+           rustup toolchain install nightly -c rustc
```

Comment:
Can we use the existing rust toolchain like we do in PyMongo instead of installing it? See https://github.com/mongodb/mongo-python-driver/blob/a4c90ae157ffcb1d4a073ceeb9177400126ad871/.evergreen/config.yml#L77-L78

Reply:
There was no `/home/admin`, nor `/home/ubuntu/.cargo` on the host, so I followed what is done in [`bson-rust`](https://github.com/mongodb/bson-rust/blob/79ab64ec484124730027c7a6cb84dd33ea0114f1/.evergreen/install-dependencies.sh#L4).

Reply:
Okay then SGTM.